### PR TITLE
Update git2 to 0.19.0

### PR DESCRIPTION
**Pull request by roele**
Update dependency git2 to latest version.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ edition = "2021"
 cargo-lock = { version = "9.0", optional = true, default-features = false }
 semver = { version = "1.0", optional = true }
 chrono = { version = "0.4", optional = true, default-features = false, features = ["clock"] }
-git2 = { version = "0.18", optional = true, default-features = false, features = [] }
+git2 = { version = "0.19.0", optional = true, default-features = false, features = [] }
```
Comment (lukaslueg) [inline Cargo.toml:17]: Please change this to `0.19` (instead of `0.19.0`)

**Review by lukaslueg — CHANGES_REQUESTED** [review]

**Comment by lukaslueg** [comment]
Thanks!

**Comment by lukaslueg** [comment]
@roele Pushed 0.7.4

**Comment by roele** [comment]
@lukaslueg Thanks a lot for the quick turnaround!

**Pull request merged**